internal/lifecycle/readme.go

```diff
@@ -79,17 +79,17 @@ func (cb codeBlock) toCommands(serviceName, gcrURL string) ([]*exec.Cmd, error)
 			line = line + l
 		}
 
-		line = os.ExpandEnv(line)
 		line = gcrURLRegexp.ReplaceAllString(line, gcrURL)
 		line = replaceServiceName(line, serviceName)
-		sp := strings.Split(line, " ")
 
 		var cmd *exec.Cmd
-		if sp[0] == "gcloud" {
-			a := append(util.GcloudCommonFlags, sp[1:]...)
-			cmd = exec.Command("gcloud", a...)
+		if gcloudCommandRegexp.MatchString(line) {
+			sp := strings.Split(line, " ")
```

Comment:
This might be an issue in the future - different systems have different split rules, so you may want to find a library that splits the same way bash does. You can put a TODO here for now and open an issue to track if you want.

Reply:
Okay, thank you, opening an issue for this and adding a comment in the code.